lib/wpscan/wp_target/wp_custom_directories.rb

```diff
@@ -26,7 +26,7 @@ def default_wp_content_dir_exists?
 
       if WpTarget.valid_response_codes.include?(response.code)
         hash = WebSite.page_hash(response)
-        return true if hash != error_404_hash and hash != homepage_hash
+        return true if hash != site_404_hash and hash != homepage_hash
```

Comment:
why don't you check against the hash set here?

Reply:
As I mentioned above, calling `error_404_hash_set` method here leads to stackoverflow.